modules/virt-single-node-cluster.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-single-node-cluster_{context}"]
+= Changes in {VirtProductName} behavior on a single node cluster
+
+By default, you can use {VirtProductName} with single node clusters, also known as xref:../../operators/operator_sdk/osdk-ha-sno.adoc#osdk-ha-sno[Single Node OpenShift (SNO)]. However, these clusters are not configured for high-availability operation, which results in significant changes to the behavior of the cluster:
+
+* {VirtProductName} components have only one replica, and `PodDisruptionBudgets` are skipped.
+* Live migration is not available. There is no supported way to migrate a virtual machine to the same node that it is already on.
+* Due to differences in storage behavior, some virtual machine templates are incompatible with SNO. To ensure compatibility, you must remove the `evictionStrategy` from any templates or virtual machines that use data volumes or storage profiles.
```

Comment:
If this is a reference to the `evictionStrategy field in the API object, I would suggest rewording the second sentence as:

```
To ensure compatibility, do not set the `evictionStrategy` field for any templates or virtual machines that use data volumes or storage profiles.
```